Frends.Salesforce.CreateSObject/Frends.Salesforce.CreateSObject.Tests/UnitTests.cs

```diff
@@ -0,0 +1,292 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+using RestSharp;
+using static Frends.Salesforce.CreateSObject.Definitions.Enums;
+using JsonSerializer = System.Text.Json.JsonSerializer;
+
+namespace Frends.Salesforce.CreateSObject.Tests
```

Comment:
Namespace should be defined using one-line definition, so `namespace Frends.Salesforce.CreateSObject.Tests;`.